io.github.appoutlet.GameOutlet.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.appoutlet.GameOutlet
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: GameOutlet
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  - --device=all
+  - --socket=x11
+  - --socket=wayland
```

Comment:
if it supports wayland then line 10 is `fallback-x11`. if it doesn't, then line 11 should be removed.